Studies/source/MvaTraining.cxx

```diff
@@ -420,55 +415,47 @@ class MVATraining {
         std::cout<<"kolmogorov"<<std::endl;
         std::map<std::string, std::map<SampleId,double>> kolmogorov = Kolmogorov(evaluation);
         std::cout<<"Significativity"<<std::endl;
-        std::map<std::string, std::pair<double, double>> sign = Significativity(outputBDT);
+        std::map<std::string, std::pair<double, double>> sign = EstimateSignificativity(outputBDT);
 
         ntuple::MvaTuple mva_tuple(outfile.get(), false);
-        auto n_trees_option = options.at("NTrees");
-        auto shrinkage_option = options.at("shrinkage");
-        auto BaggedSampleFraction_option = options.at("BaggedSampleFraction");
-        auto MaxDepth_option = options.at("MaxDepth");
-        auto MinNodeSize_option = options.at("MinNodeSize");
+        const auto& n_trees_option = options.at("NTrees");
+//        const auto& shrinkage_option = options.at("shrinkage");
```

Comment:
instead of commenting/uncommenting code every time you change config, use pointers